docs/services/sdk/serving.mdx

```diff
@@ -0,0 +1,178 @@
+---
+sidebar_position: 9
+description: "Set up long-running services or Lambda handlers."
+---
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Serving
+Restate services can run in two ways: as long-running services or as AWS Lambda functions.
+
+## Restate server for long-running services
+
+To register your service as a long-running service, do the following:
+
+<Tabs groupId="ts-api">
+<TabItem value="handler" label="Handler API" default>
+
+**1. Define a router for your keyed or unkeyed service**
+
+This router is used to link the function you implemented with an API path. 
+
+For **unkeyed services**, the router looks as follows:
+
+```typescript
+const serviceRouter = restate.router({
+    hello: async () => { ... },
+    callMe: async (ctx: restate.RpcContext) => { ... },
+    maybe: async (ctx: restate.RpcContext, request: Request) => { ... }
+});
+```
+
+This example shows a router for an unkeyed service with three functions and their respective API paths (`hello`, `callMe`, and `maybe`).
+This means that the `hello` function can be called via `<restate-host>:9090/myServiceName/hello`.
+
+The functions of an unkeyed router have two optional parameters:
+
+- A `ctx` of type `restate.RpcContext` which allows to interact with Restate.
+- A `request` of type JSON object that represents additional invocation data.
+
+The parameter names may be chosen differently.
+
+For **keyed services**, the router looks as follows:
+```typescript
+// 2. Create a router for keyed services (if any):
+const keyedServiceRouter = restate.keyedRouter({
+    hello: async () => { ... },
+    callMe: async (ctx: restate.RpcContext) => { ... },
+    maybe: async (ctx: restate.RpcContext, key: string) => { ... },
+    withSomething: async (ctx: restate.RpcContext, key: string, request: Request) => { ... }
+});
+```
+
+This example shows a router for a keyed service with four functions and their respective API paths (`hello`, `callMe`, `maybe`, and `withSomething`).
+This means that the `hello` function can be called via `<restate-host>:9090/myServicePath/hello`.
```

Comment:
Above it is `myServiceName` and here it is `myServicePath`. Should probably be harmonized.